manifests/charts/cloudcore/templates/rbac_controllermanager.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if (.Values.controllerManager.enable) }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: controller-manager
+  namespace: {{ .Release.Namespace }}
+  {{- with .Values.controllerManager.labels }}
+  labels: {{- toYaml . | nindent 4 }}
+  {{- end }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: controller-manager
+  {{- with .Values.controllerManager.labels }}
+  labels: {{- toYaml . | nindent 4 }}
+  {{- end }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: controller-manager
+subjects:
+  - kind: ServiceAccount
+    name: controller-manager
+    namespace: kubeedge
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: controller-manager
+  {{- with .Values.controllerManager.labels }}
+  labels: {{- toYaml . | nindent 4 }}
+  {{- end }}
+rules:
+  - apiGroups: ["apps.kubeedge.io"]
+    resources: ["nodegroups", "nodegroups/status"]
+    verbs: ["*"]
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["list", "watch", "patch", "get"]
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["list", "watch", "get", "delete"]
```

Comment:
do we need delete pod?

Reply:
Oh, actually it does not delete the pod. I'll revise it.

Reply:
Sorry for my mistake, the authorization for deleting pod is needed for nodegroup controller. I'll explain the scenario:

Considering that we have an eapp deployed in the cluster which having pods running on `node-a` and `node-b`, and `node-a` `node-b` are belonging to `nodegroup-hangzhou`, when we remove `node-a` from `nodegroup-hangzhou`, we should evict the pods of eapp running on `node-a`, and then create new instances on `node-b` or other nodes in `nodegroup-hangzhou`.   The eviction needs to delete the pod.

Reply:
I think we should not do this is nodegroup controller, In edgeapplication, we manage resouces such as deployment， statefulset and so on, we donot manage pods directly. when user delete a pod,  he should do this carefully as below
https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/safely-drain-node/
when he drain the node, all pod on this node will be evict. and the k8s kube-controller will watch this and then create new pod for the deployment or statefulset.

Reply:
there will be conflict when we mange pod vs k8s controller manager

Reply:
You're right when deleting a node from cluster. But other scenarios we should take into consideration. Let's continue with the above example:  
Situation 1: `node-a` is not deleted from cluster, but just be an orphan node, which means it does not belong to any nodegroup.
Situation 2: `node-a` is moved from `nodegroup-hangzhou` to `nodegroup-beijing`.

The original idea of eviciting pods on node is to handle the above 2 situations, but the situation you mentioned above seems also be included. I need to figure out if there's competation between nodegroup controller and k8s controller manager when deleting nodes.